user-agreement.md

```diff
@@ -31,11 +31,11 @@ To clarify, "**FriendlyU Content**" means the text, data, graphics, images, phot
 
```

Comment:
The Site, Services, and FriendlyU Content are intended solely for persons who are 13 or older. Any access to or use of the Site, Service, and FriendlyU Content by anyone under 13 is strictly prohibited. By using the Site and Service, and FriendlyU Content you represent and warrant that you are 13 or older.